docs/source/concepts/FractionalRebinning.rst

```diff
@@ -0,0 +1,170 @@
+.. _FractionalRebinning:
+
+================================================
+Fractional Rebinning (RebinnedOutput Workspaces)
+================================================
+
+.. contents::
+  :local:
+
+In some instances of rebinning data, the information of the counts in the bins
+prior rebinning should be preserved in the output of rebinning in some way.
+This is because some algorithms depend on how the original data was treated.
+We achieve this by storing a weight for each bin that corresponds to the fraction
+between the area of the output bin and the area of the input bin (i.e. prior to rebinning).
+This kind of **Workspace2D** that contains information of the areas used during rebinning
+is called a **RebinnedOutput**. Some algorithms, such as
+:ref:`SofQWNormalisedPolygon <algm-SofQWNormalisedPolygon>`
+and :ref:`ConvertToReflectometryQ <algm-ConvertToReflectometryQ>`, create this special type of
+:ref:`Workspace2D <Workspace2D>` workspace in which
+each bin contains both a value and the fractional overlap area of the this bin over
+that of the original data. This fractional overlap area value is referred to as
+a fractional weight, since it has the role of controlling the proportion of the counts
+in output bins that underwent some coordinate transformation.
+
+To illustrate why using these fractional weights is necessary we explain how rebinning works
+through the technique of *NormalisedPolygon*.
+
+NormalisedPolygon Technique
+---------------------------
+
+*NormalisedPolygon* constructs
+a polygon using the boundaries of the input bin, and transforms that polygon
+into the output coordinates. In the output coordinates, we look for the intersections
+between the transformed input bin and the output bin:
```

Comment:
I think maybe a bit more introduction is needed here. The reason that the input and output grids are not parallel is that the `SofQWNormalisedPolygon` algorithm does not simply do a rebinning within the same coordinate system - rather it does: 

1. A coordinate transformation (from scattering angle `theta` to momentum transfer `Q`).
2. A (fractional) rebinning in the new coordinate system.

Now, the fractional rebinning could be applied even if there was no coordinate transformation (this is what `Rebin2D` does) but the figure with the parallelpiped only applies if there has been a coordinate transformation.